include/boost/url/params.hpp

```diff
@@ -71,7 +71,7 @@ class params
     /** A random-access iterator referencing parameters in a url query.
 
         This iterator lazily constructs instances of @ref params::reference, which
-        contain @ref const_string decoded representations of the current <key, value> pair
+        contain @ref const_string decoded representations of the current key-value pair
         in the underlying @ref url.
 
         The @ref params underlying allocator is used to construct the instances
```

Comment:
This reads funny. I don't think you should say `@ref params` (does that even work)? Just say that memory for allocated strings uses the allocator specified on construction of the container.